weatherreport.js

```diff
@@ -9,7 +9,7 @@ const weatherSensorStub = {
     humidity: () => 72,
     precipitation: () => 70,
     temperatureInC: () => 26,
-    windspeedInKmph: () => 52,

```

Comment:
With this change, we are changing it to test low wind speed. The condition of high wind speed is not tested anymore. Actually now `testRainy` and `testHighPrecipitationAndLowWindspeed` both test the same thing.
We want to test our function at different weathers, without actually waiting for the weather to change. Can you change it such that the two tests will test different weather conditions?